### PR TITLE
pyroute2: Added aarch64 support

### PR DESCRIPTION
**Pull request by ossdev07**
This patch adds support for the aarch64 architecture.

### CODE REVIEW

**Comment by ossdev07** [comment]
Ping @svinota could you please have a look at this PR.

**Comment by ossdev07** [comment]
@svinota Please, could you share your feedback on this patch?

**Comment by etene** [comment]
Hello,

As you can see on his contribution graph, @svinota is often "offline" for a few weeks at a time, and we can't blame him for it. It seems we are currently in such a period.

I'm sure he'll take a look at your patch when he gets back :)

File: pyroute2/netlink/rtnl/ifinfmsg/proxy.py [hunk 1]
```diff
@@ -27,7 +27,8 @@
 IFNAMSIZ = 16
 
 TUNDEV = '/dev/net/tun'
-if config.machine in ('i386', 'i686', 'x86_64', 'armv6l', 'armv7l', 's390x'):
+PLATFORMS = ['i386', 'i686', 'x86_64', 'armv61', 'armv7l', 's390x', 'aarch64']
```
Comment (Vic063) [inline pyroute2/netlink/rtnl/ifinfmsg/proxy.py]: Warning: you changed 'armv6l' to 'armv61'

[the remark this answers is not in the document]
Comment (ossdev07) [reply pyroute2/netlink/rtnl/ifinfmsg/proxy.py]: Implemented the changes mentioned by @ffourcot and @Vic063. Please have a look.

**Comment by ffourcot** [comment]
@ossdev07 I only see a change in commit message, not in the code. Did your forgot to push something?

**Comment by ossdev07** [comment]
@ffourcot I have committed again. Please have a look.

**Comment by ffourcot** [comment]
Merged, thanks!

**Pull request closed without merging**